config/config-api/src/com/thoughtworks/go/config/PipelineConfig.java

```diff
@@ -164,7 +168,11 @@ private void validateStageNameUniqueness() {
 
     private void validateLabelTemplate() {
         if (XmlUtils.doesNotMatchUsingXsdRegex(LABEL_TEMPLATE_FORMAT_REGEX, labelTemplate)) {
-            addError("labelTemplate", "Invalid label. Label should be composed of alphanumeric text, it should contain material revision or ${COUNT}, or use params as #{<param-name>}.");
+            addError("labelTemplate",
```

Comment:
To the best of my knowledge this is the message with is shown in UI when I edit a pipeline label in the the pipeline admin form (rather than the settings.xml admin form). 

Hence it should be expressive, and I do not have a clue how to make it shorter and have it still explaining enough. What about pointing to a documentation page or help text?